llvm/test/CodeGen/AMDGPU/GlobalISel/fmamix-constant-bus-violation.ll

```diff
@@ -0,0 +1,23 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -global-isel -mtriple=amdgcn -mcpu=gfx908 -denormal-fp-math=preserve-sign -verify-machineinstrs %s -o - | FileCheck %s
```

Comment:
Don't need -verify-machininstrs. Also prefer to just use the explicit denormal-fp-math-f32 attribute in the test itself